apps/admin/app/(auth)/register/_components/form.tsx

```diff
@@ -21,8 +28,44 @@ export function RegisterationForm() {
     },
   });
 
-  const submitfunction = () => {
-    console.log("Submitted");
+  const submitFunction = () => {
+    const data = {
+      email: form.getValues("email"),
+      password: form.getValues("password"),
+      name: `${form.getValues("firstName")} ${form.getValues("lastName")}`,
+    };
+
+    axios
+      .post<TRegistrationSuccessResponse>(REGISTER_URL, data)
+      .then((response) => {
+        document.cookie = `Authorization=${response.data.token.access}`;
+        document.cookie = `Refresh=${response.data.token.refresh}`;
+        router.push("/dashboard");
+      })
+      .catch((error: AxiosError<TRegistrationErrorResponse>) => {
+        if (error.response) {
+          error.response.data.email &&
```

Comment:
```suggestion
         const error =  error.response.data
         error.email && form.setError("email", {
              type: "server",
              message: error.email[0],
            });
         ```
         is it possible to reduce the size of this function

Reply:
For the time being until we come up with a consistent naming scheme for frontend and backend, it is not possible to reduce the function size